### DISCUSSION TITLE
Open Resource Edit in new Tab

### DISCUSSION THREAD

**Discussion by gepopp** [laravel/nova-issues#5165] [Q&A] [answered]
I dont know with which update it happens, but the view and edit icons in a resource index page seem to be buttons now, not links anymore.

This comes quite unhandy when u wanna edit multiple resources in a row bcs its not possible to do a right click and open the edit page in a new tab. Now after editing you gotta go back to index, paginate to the right page and find the resouce an so on....

A setting that opens the resource edit page in a new tab would be a really great timesaving feature!

**Comment by crynobone** [comment]
Are you running the latest version?

**Reply by gepopp** [reply]
Yes i do.

I have another project that is on version 4.16, there the edit and view Icons are rendered inside an anchor tag.
On the current version it's inside a button.

**Comment by crynobone** [comment, marked as answer]
The issue should already been fixed in `4.19.5`: https://nova.laravel.com/releases/4.19.5

Please make sure you have the updated asset by running `php artisan nova:publish`

**Reply by gepopp** [reply]
a damn it, it was on 4.19.4, im sorry ive must have missed that last update.

Thanks for the reply, made my day!!!
